README.md

```diff
@@ -190,6 +190,20 @@ If a job fails more than `max_retry` times, the payload is sent to `jobs.[job_na
 
 Under the hood, TaskBunny uses [dead letter exchanges](https://www.rabbitmq.com/dlx.html) to support retry.
```

Comment:
Yes, TaskBunny still uses dead letter exchange to move messages from retry queue to work queue.